external_parser/joiners/example_joiner.cc

```diff
@@ -498,7 +498,11 @@ bool example_joiner::process_joined(v_array<example *> &examples) {
     return true;
   }
 
-  je->fill_in_label(examples);
+  if (!je->fill_in_label(examples)) {
+    _current_je_is_skip_learn = true;
```

Comment:
Is this skip learn or is this a bug in the data that should be reported (i.e. if you don't set this to true then we will continue learning but this will get logged as an error [here](https://github.com/VowpalWabbit/reinforcement_learning/blob/master/external_parser/parse_example_binary.cc#L268)

Reply:
yeah you are right, it should be reported. let me fix that.

Reply:
fixed